### PR TITLE
fix multi dataplane related issues

### PR DESCRIPTION
**Pull request by tharindu1st**
fix multi dataplane related issues

### CODE REVIEW

File: helm-charts/templates/cert-manager/issuers/apk-root-certificate-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
...QkFRRUZBQU9DQVE4QU1JSUJDZ0tDCkFRRUExRzlCQWNWYTNhTWVteEVSWStKOFVtSC9XOEpwY0VPdGRlTlg1WU5paG5OT1h0bHZoU3pGek9QYUs5N1AKcjRJcUdBU2JWTmlSKzFKNldFb2kvYjZaSlR4MHEzWVVuMFlsUUpyejdnMjBUZG9HSmp4R1ZXem4wRVc0YmVIWApHcTYwdlhMZjR0M21sTENMR0lLM2tKVFdBb1J6ZDc0ZGpWNys1djBCbS82S0JCQVdjdTVVYk9EOUtScE9zeEdNCm4zWjAxMDNvQUdWaXlxODRRdEZ2aFhWTld0dERMZTJqVS83bzQyZGRhSm96Ukw5eisxQWVwZG9XUHlKWklacVUKYlhjR0FrN2lkazdjLzhkS014d0FtM0NWL1d2Z1dyVks1UitZVGlHcVJmNXBkOVdXQ3lkRVZRa05xQ1pnVFBOeQpCVFJ2SG81Mm9uUG5UNkFMdE1JMG1uV0x0UUlEQVFBQm8xTXdVVEFkQmdOVkhRNEVGZ1FVenhjQThjZUNGNXQrCnZQZU9wWWJpMTFDV2p3Y3dId1lEVlIwakJCZ3dGb0FVenhjQThjZUNGNXQrdlBlT3BZYmkxMUNXandjd0R3WUQKVlIwVEFRSC9CQVV3QXdFQi96QU5CZ2txaGtpRzl3MEJBUXNGQUFPQ0FRRUFGQ0tkNXg4ejY0cDFqOUJub1NBbApKSnVOejR1V2p2L1lLOTRCM3MzS1VrWE9Yakh1THVhemtTc3MyVUhGYitLZUxkNnZoaW1NNVFxa0ZzVUVNc25DCjhSeUJaUDU4a2doY0x6SmxkNVVpd2xuci9SdU9BTnZjdjRlS1Nhdnd1NS9BQlh1TVVRYi9Hdkt0V1BycjJWYkoKVUxnM3A3TkdpZ1hISGc4NGVWTUE3b05YMVo1UjJjUzRJU2tsV1htNVNwTVBoK1NOQ2dxd3FoeFJOWUoySjBFWgpxbHA0b2ZRRzNHSjcySitEUkhsTnVqV0Vza1A1SUpqdzZ3OFEwempYeDI2eWVsR2UyK1RNNkJCN1BwQ042a05VCnpIbzJrLzU3NWJ1MmlaenRuWVZtRTc0SDFXM2NYSjdjMHE4MnVVRnZkVytGbFJ0bStPUElpSUdLNzRsRmlaTkIKT1E9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```
Comment (Amila-Rukshan) [inline helm-charts/templates/cert-manager/issuers/apk-root-certificate-secret.yaml:10]: shall we add other issuer properties for this CA considering wso2 org properties?
```
    "issuer": {
        "C": "US",
        "ST": "CA",
        "L": "Mountain View",
        "O": "WSO2",
        "OU": "WSO2",
    }
```

File: helm-charts/templates/control-plane/management-server/management-server-deployment.yaml [hunk 2]
```diff
@@ -59,15 +75,26 @@ spec:
...
           {{- else }}
-            secretName: {{ template "apk-helm.resource.prefix" . }}-adapter-server-cert
+            secretName: {{ template "apk-helm.resource.prefix" . }}-management-server-cert
           {{- end }}
+        - name: adapter-truststore-secret-volume
+          secret:
+          {{ if and .Values.wso2.apk.dp.adapter.configs .Values.wso2.apk.dp.adapter.configs.tls }}
+            secretName: {{ .Values.wso2.apk.dp.adapter.configs.tls.certificatesSecret | default "apk-root-certificate"}}
+          {{ else }}
+            secretName: apk-root-certificate
```
Comment (Amila-Rukshan) [inline helm-charts/templates/control-plane/management-server/management-server-deployment.yaml:89]: this root certificate for validating certs when connecting to multiple adaptors?

File: helm-charts/values.yaml [hunk 3]
```diff
@@ -168,8 +168,8 @@ wso2:
               cpu: "1000m"
           strategy: Recreate
           replicas: 1
-          imagePullPolicy: Always
-          image: wso2/management-server:latest
+          imagePullPolicy: IfNotPresent
+          image: management-server:0.0.1-SNAPSHOT
```
Comment (Amila-Rukshan) [inline helm-charts/values.yaml:172]: please remove these temporary changes

File: helm-charts/templates/control-plane/management-server/management-server-deployment.yaml (hunk 2 above)
Comment (tharindu1st) [reply helm-charts/templates/control-plane/management-server/management-server-deployment.yaml:89]: yes

**Review by Krishanx92 — APPROVED** [review]

**Review by Amila-Rukshan — APPROVED** [review]

**Pull request merged**